specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/examples/integrations/DeleteSecurityConnectorIntegration_example.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
+    "api-version": "2023-07-01-preview",
+    "scope": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/gcpResourceGroup/providers/Microsoft.Security/securityConnectors/gcpconnector",
+    "integrationId": "ad9a8e26-29d9-4829-bb30-e597a58cdbb8"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/gcpResourceGroup/providers/Microsoft.Security/securityConnectors/gcpconnector/integrationId/ad9a8e26-29d9-4829-bb30-e597a58cdbb8?api-version=2024-01-31"
```

Comment:
Does the intent is to use 'integrations' resource as an extension resource here? the location should signify that - i.e. - https://management.azure.com/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/gcpResourceGroup/providers/Microsoft.Security/securityConnectors/gcpconnector/providers/Microsoft.Security/integrations/ad9a8e26-29d9-4829-bb30-e597a58cdbb8?api-version=2023-07-01-preview